README.md

```diff
@@ -36,36 +34,39 @@
 
 ## About
 
-[Read more about BUX](https://getbux.io)
+Complete stand-alone server using the SPV Wallet engine (UTXOs, xPubs, Paymail & More!)
```

Comment:
Perhaps consider phrasing it as: `A comprehensive non-custodial wallet for BSV digital currency.` 
I would suggest omitting the `using (...) Engine` portion, particularly since we plan to integrate bux (engine) into bux-server. After this integration, the concept of a separate, distinguishable engine will no longer apply.